### PR TITLE
Bdd/bullseye and custom rules

### PR DESCRIPTION
**Pull request by jmecosta**
striped down to only needed files for me to create new data.

taken out the build look up tables from xunit sensor in case filename tags are found

### CODE REVIEW

**Comment by jmecosta** [comment]
i would like to create a bdd for multimodule, but would like this one to be merged so i can use the same feature and reuse the test data...

**Comment by jmecosta** [comment]
rebased

File: integration-tests/features/test_create_rules_and_bullseye_import.feature [hunk 1]
```diff
@@ -0,0 +1,65 @@
+Feature: GoogleTestWithBullseyeAndVsProject
+
+  This test verifies that analysis is able to import bullseye coverage reports and import custom rules reports
```
Comment (guwirth) [inline integration-tests/features/test_create_rules_and_bullseye_import.feature]: Could you add a more detailed description what the test is doing please. Makes it easier for others to understand and extend it. THX.

**Comment by guwirth** [comment]
PR looks good for me.

File: integration-tests/features/test_create_rules_and_bullseye_import.feature (hunk 1 above)
Comment (jmecosta) [inline integration-tests/features/test_create_rules_and_bullseye_import.feature]: Will do

**Pull request merged**